docs/_static/spec/server.yaml

```diff
@@ -0,0 +1,1517 @@
+    conversation_id: default
+    latest_event_time: 1537645578.314389
+    latest_input_channel: rest
+    paused: false
+    slots: {}
+    followup_action:
+    latest_message:
+      text: hello
+      entities: []
+      intent:
+        confidence: 0.57
+        name: greet
+      intent_ranking:
+      - confidence: 0.57
+        name: greet
+      - confidence: 0.33
+        name: goodbye
+    events:
+    - event: action
+      name: action_listen
+      timestamp: 1537645578.274815
+    - event: user
+      input_channel: rest
+      text: hello
+      timestamp: 1537645578.314389
+      parse_data:
+        text: hello
+        entities: []
+        intent:
+          confidence: 0.57
+          name: greet
+        intent_ranking:
+        - confidence: 0.57
+          name: greet
+        - confidence: 0.33
+          name: goodbye
+
+paths:
+  /:
+    get:
+      tags:
+      - Server Information
+      summary: Health endpoint of Rasa Server
+      description: >-
+        This URL can be used as an endpoint to run
+        health checks against. When the server is running
+        this will return 200.
+      responses:
+        200:
+          description: Up and running
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: Welcome text of Rasa Server
+              example: >-
+                Hello from Rasa: 0.12.0
+
+  /version:
+    get:
+      security:
+      - TokenAuth: []
+      - JWT: []
+      tags:
+      - Server Information
+      summary: Version of Rasa
+      description: >-
+        Returns the version of Rasa.
+      responses:
+        200:
+          description: Version of Rasa
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  version:
+                    type: string
+                    description: >-
+                      Core version number
+                  minimum_compatible_version:
+                    type: string
+                    description: >-
+                      Minimum version this Rasa version is
+                      able to load models from
+              example:
+                version: 0.12.0
+                minimum_compatible_version: 0.12.0
+        401:
+          $ref: '#/components/responses/401NotAuthenticated'
+        403:
+          $ref: '#/components/responses/403NotAuthorized'
+
+  /status:
+    get:
+      security:
+      - TokenAuth: []
+      - JWT: []
+      tags:
+      - Server Information
+      summary: Status of the currently loaded Rasa Core & NLU models
+      description: >-
+        Information about the currently loaded Rasa Core & NLU models.
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+              example:
+                  core:
```

Comment:
I think this needs some different content, e.g. `fingerprint.json` from the model archive?

Reply:
Changed the response to the content of the `fingerprint.json` file plus the model name (e.g. file name).